### PR TITLE
Retry create database in InfluxDbIOIT

### PR DESCRIPTION
**Pull request by Abacn**
This should resolve the initial issue raised in #23286 (though https://github.com/apache/beam/issues/23286#issuecomment-1341548232 needs further investigation)

**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by Abacn** [comment]
Run Java InfluxDbIO Performance Test

**Comment by Abacn** [comment]
Well, apparently "influxdbio performance test" is not a true performance test. It is an integration test running on direct runner actually.

**Comment by Abacn** [comment]
Nevertheless this test appears to be the only job exercises InfluxDBIOIT (InfluxDb_IO_Direct only runs unit tests). Preserve it and rename as PostCommit

**Comment by Abacn** [comment]
run seed job

**Comment by Abacn** [comment]
Run Java InfluxDbIO_IT

**Comment by Abacn** [comment]
Test passed: https://ci-beam.apache.org/job/beam_PostCommit_Java_InfluxDbIO_IT_PR/1/

**Comment by Abacn** [comment]
ready for review now.

**Comment by Abacn** [comment]
R: @kileys

File: .test-infra/jenkins/README.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ Beam Jenkins overview page: [link](https://ci-beam.apache.org/)
...
 | beam_PreCommit_Java_Neo4j_IO_Direct | [commit](https://ci-beam.apache.org/job/beam_PreCommit_Java_Neo4j_IO_Direct_Commit/), [cron](https://ci-beam.apache.org/job/beam_PreCommit_Java_Neo4j_IO_Direct_Cron/), [phrase](https://ci-beam.apache.org/job/beam_PreCommit_Java_Neo4j_IO_Direct_Phrase/) | `Run Java_Neo4j_IO_Direct PreCommit` | [![Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Java_Neo4j_IO_Direct_Cron/badge/icon)](https://ci-beam.apache.org/job/beam_PreCommit_Java_Neo4j_IO_Direct_Cron/) |
+| beam_PreCommit_Java_SingleStore_IO_Direct | [commit](https://ci-beam.apache.org/job/beam_PreCommit_Java_SingleStore_IO_Direct_Commit/), [cron](https://ci-beam.apache.org/job/beam_PreCommit_Java_SingleStore_IO_Direct_Cron/), [phrase](https://ci-beam.apache.org/job/beam_PreCommit_Java_SingleStore_IO_Direct_Phrase/) | `Run Java_SingleStore_IO_Direct PreCommit` | [![Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Java_SingleStore_IO_Direct_Cron/badge/icon)](https://ci-beam.apache.org/job/beam_PreCommit_Java_SingleStore_IO_Direct_Cron/) |
```
Comment (ahmedabu98) [inline .test-infra/jenkins/README.md:44]: Thanks for adding SingleStoreIO tests here too

CC: @AdalbertMemSQL

**Review by ahmedabu98 — APPROVED** [review]
LGTM

**Pull request merged**